templates/libreswan-host-to-host.secrets.j2

```diff
@@ -6,12 +6,8 @@
 {%       if host == inventory_hostname or host == ansible_host %}
 {%         for otherhost, otherval in __vpn_psks[__vpn_idx][host].items() %}
 {%           if otherhost == item.item %}
-{%             if tunnel.hosts[otherhost] is mapping and 'hostname' in tunnel.hosts[otherhost] %}
-{%               set otherhost = tunnel.hosts[otherhost].hostname %}
-{%             endif %}
-{%             if tunnel.hosts[host] is mapping and 'hostname' in tunnel.hosts[host] %}
-{%               set host = tunnel.hosts[host].hostname %}
-{%             endif %}
+{%             set host =  tunnel.hosts[host].hostname | d((hostvars[host] | d({})).ansible_host | d(host)) %}
+{%             set otherhost =  tunnel.hosts[otherhost].hostname | d((hostvars[otherhost] | d({})).ansible_host | d(otherhost)) %}
```

Comment:
extra space here

Reply:
sorry about that, thought I got all those. Should be good now